connector_jira_servicedesk/models/project_project/project_link_jira.py

```diff
@@ -22,6 +22,7 @@ def state_exit_start(self):
             self.state = "link_organizations"
         else:
             super().state_exit_start()
+        return
```

Comment:
@sonhd91 we don't need this change

Reply:
pre-commit raise should use return because super is used

Reply:
how about `return super().state_exit_start()`?